resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -0,0 +1,242 @@
+                for fq_report in d.fastqc_archive:
+                    name = os.path.basename(fq_report.path)
+                    create_symlink(fq_report.path, os.path.join(sample_dir, name))
+
+            elif d.type.startswith('data:reads:fastq:paired'):
+                for fq_report in d.fastqc_archive + d.fastqc_archive2:
+                    name = os.path.basename(fq_report.path)
+                    create_symlink(fq_report.path, os.path.join(sample_dir, name))
+
+            elif d.type == 'data:alignment:bam:star:':
+                stats_file = os.path.basename(d.stats.path)
+                assert stats_file.endswith('_stats.txt')
+                bam_name = stats_file[:-10]
+
+                if d.build == 'rRNA':
+                    rrna_report = f'{bam_name}.rRNA.Log.final.out'
+                    create_symlink(d.stats.path, os.path.join(sample_dir, rrna_report))
+                elif d.build == 'globin':
+                    globin_report = f'{bam_name}.globin.Log.final.out'
+                    create_symlink(d.stats.path, os.path.join(sample_dir, globin_report))
+                else:
+                    report = f'{bam_name}.Log.final.out'
+                    create_symlink(d.stats.path, os.path.join(sample_dir, report))
+
+            elif d.type.startswith('data:alignment:bam'):
+                name = os.path.basename(d.stats.path)
+                create_symlink(d.stats.path, os.path.join(sample_dir, name))
+
+            elif d.type == 'data:expression:featurecounts:':
+                name = os.path.basename(d.counts_summary.path)
+                create_symlink(d.counts_summary.path, os.path.join(sample_dir, name))
+
+            elif d.type == 'data:chipseq:callpeak:macs2:':
+                name = os.path.basename(d.called_peaks.path)
+                create_symlink(d.called_peaks.path, os.path.join(sample_dir, name))
+
+            elif d.type == 'data:samtools:idxstats:':
+                name = os.path.basename(d.report.path)
+                create_symlink(d.report.path, os.path.join(sample_dir, name))
+
+            elif d.type == 'data:qorts:qc:':
+                qorts_path = os.path.join(sample_dir, 'QoRTs')
+                os.mkdir(qorts_path)
+                name = os.path.basename(d.summary.path)
+                create_symlink(d.summary.path, os.path.join(qorts_path, name))
+
+            elif d.type == 'data:expression:salmon:':
+                create_symlink(d.salmon_output.path, sample_dir)
+
+            elif d.type.startswith('data:alleyoop'):
+                create_symlink(d.report.path, sample_dir)
+                # Alleyoop summary may contain PCA plot data
+                if os.path.isfile(d.plot_data.path):
+                    create_symlink(d.plot_data.path, sample_dir)
+
+            else:
+                unsupported_data.append(d.id)
+
+        if unsupported_data:
+            self.warning(f'Data with ID(s) {unsupported_data} is not supported by the MultiQC analysis.')
```

Comment:
Couldn't this be moved to line 211 under the final `else`?

```
else:
    unsupported_data.append(d.id)
    self.warning(f'Data with ID(s) {unsupported_data} is not supported by the MultiQC analysis.')
```

I think you can remove "is" to read 

```
self.warning(f'Data with ID(s) {unsupported_data} not supported by the MultiQC analysis.')
```

Reply:
Moving this to final `else` would issue as many warnings as there is unsupported Data. In practice this was unreadable in many cases, thus the change to issue just one warning containing a list of unsupported Data IDs.